ntbs-service/Models/ClinicalDetails.cs

```diff
@@ -30,5 +30,14 @@ public class ClinicalDetails
 
         public Status BCGVaccinationState { get; set; }
         public int? BCGVaccinationYear { get; set; }
+
+        [OnlyOneTrue("IsMDRTreatment", ErrorMessage = ValidationMessages.ValidTreatmentOptions)]

```

Comment:
Curious if we could use the library that Farman is adding in the validation ticket to achieve this with something along the lines of 
`Assert["IsMDRTreatment == null"]` ..?
Not worth spending lots of time, but if Farman's branch merges in beforehand it might be worth a 5min test to check that would work, and if not leaving a comment to the effect? Less custom code, the better! :)

Reply:
Yes, with `Assert["IsMDRTreatment == null"]` should do the job. It's applied only if the value is not null

Reply:
Will check it out when I merge :)